### DISCUSSION TITLE
ptxas not engaged

### DISCUSSION THREAD

**Discussion by SophiaUCL** [bonsai-rx/bonsai#2053] [Q&A]
Hi, I get the following message when I try to run Bonsai: 

2024-11-12 09:47:06.100475: W tensorflow/compiler/mlir/tools/kernel_gen/transforms/gpu_kernel_to_blob_pass.cc:189] Failed to compile generated PTX with ptxas. Falling back to compilation by driver.

I'm working on Windows and I have Tf version 2.10.1, Cuda version 11.2, Cudnn version 8.1.0.. I have added ptxas to the path environment. What could be the problem?

**Comment by bruno-f-cruz** [comment]
Is this running any particular package like Sleap or DLC?

**Comment by SophiaUCL** [comment]
It is running sleap


________________________________

**Reply by bruno-f-cruz** [reply]
Do you get this error when running inference in the Sleap Python package too?

**Comment by SophiaUCL** [comment]
No, I don't get it then


________________________________

**Comment by SophiaUCL** [comment]
I've played around a bit. This error has disappeared. It got solved by adding ptxas to the extensions folder


________________________________
